src/Sarif.Multitool/RewriteCommand.cs

```diff
@@ -39,6 +41,23 @@ public static int Run(RewriteOptions rewriteOptions)
             return 0;
         }
 
+        private static IDictionary<string, Uri> ConstructUriBaseIds(IEnumerable<string> uriBaseIds)
+        {
+            if (uriBaseIds == null) { return null; }
+
+            IDictionary<string, Uri> result = new Dictionary<string, Uri>();
```

Comment:
>result [](start = 37, length = 6)

Nit: I always suggest to Chris, when I see this, that he find another variable name, since "`result`" both is very generic and has a special meaning is SARIF. If it's me, I call it `uriBaseIdsDictionary`, but YMMV.

Reply:
yes. 'result' is a particularly poor name in a SARIF context.

---
In reply to: [214520277](https://github.com/Microsoft/sarif-sdk/pull/1062#discussion_r214520277) [](ancestors = 214520277)